frontend/lib/pages/index-page.tsx

```diff
@@ -3,43 +3,93 @@ import React from 'react';
 import Page from '../page';
 import { Link, Redirect } from 'react-router-dom';
 import Routes from '../routes';
-import { WhyMailALetterOfComplaint } from '../letter-of-complaint-common';
 import { OutboundLink } from '../google-analytics';
 import { CenteredPrimaryButtonLink } from '../buttons';
 
+const locSvg = require('../svg/letter-of-complaint.svg') as JSX.Element;
+
 export interface IndexPageProps {
   isLoggedIn: boolean;
 }
 
 export default class IndexPage extends React.Component<IndexPageProps> {
   renderLoggedOut() {
     return (
-      <Page title="Technology for Housing Justice">
-        <div className="content">
-        <h1>Need repairs in your apartment? JustFix.nyc will mail a letter to your landlord!</h1>
-        <p>
-          Sending a letter of complaint to your landlord will help document all the issues that need to be fixed.
-        </p>
-        <CenteredPrimaryButtonLink to={Routes.onboarding.latestStep} className="is-medium">
-          Get started
-        </CenteredPrimaryButtonLink>
-        <p>Already have an account? <Link to={Routes.login}>Sign in!</Link></p>
-
-        <WhyMailALetterOfComplaint heading="h2" />
+      <Page className="landing-page" title="Technology for Housing Justice">
 
-        <CenteredPrimaryButtonLink to={Routes.onboarding.latestStep} className="is-medium">
-          Get started
-        </CenteredPrimaryButtonLink>
+        <section className="hero is-light">
+          <div className="hero-body">
+            <div className="has-text-centered">
+              <div className="jf-loc-image">
+                {locSvg}
+              </div>
+              <h1 className="title is-spaced">
+                Is your landlord not responding? Take action today!
+              </h1>
+              <h2 className="subtitle">
+                JustFix.nyc is a free tool that notifies your landlord of repair issues via <b>USPS<sup>&copy;</sup> Certified Mail</b>. Everything is kept confidential and secure.
+              </h2>
+              <CenteredPrimaryButtonLink to={Routes.onboarding.latestStep} className="is-large">
+                Start my letter
+              </CenteredPrimaryButtonLink>
+              <p className="secondary-cta">Already have an account? <Link to={Routes.login}>Sign in!</Link></p>
+            </div>
+          </div>
+        </section>
 
-        <h2>About JustFix.nyc</h2>
+        <section className="section">
+          <div className="content">
+            <h1 className="has-text-centered">How It Works</h1>
+            <div className="how-it-works columns is-multiline">
+              <div className="column is-half">
+                <div className="notification">
+                  <div className="num"><h1>1</h1></div>
```

Comment:
wait wut?  is this an `<h1>` just so it looks really big?  This is bad for accessibility unfortunately, as screen readers will basically use the heading structure to form a document outline, so we generally want just one `<h1>` on the page. See the [WebAIM guidelines on Using Headings for Content Structure](https://webaim.org/techniques/semanticstructure/#contentstructure) for more details!